spec/helpers/news_tematica/application_helper_spec.rb

```diff
@@ -1,14 +1,13 @@
-# coding: UTF-8
-
 require "spec_helper"
 
-describe NewsTematica::ApplicationHelper do
+describe NewsTematica::ApplicationHelper, type: :helper do
   describe "max_caracteres_con_palabras_enteras" do
     it 'Debe devolver la frase entera si no desborda la longitud' do
-      max_caracteres_con_palabras_enteras('Hola mundo', 50).should == 'Hola mundo'
```

Comment:
I think it could use for the rails helper 'truncate' rather than create your own.

`truncate("Hola mundo", length: 7, separator: ' ')`